perf.groovy

```diff
@@ -0,0 +1,76 @@
+// Groovy Script: http://www.groovy-lang.org/syntax.html
+// Jenkins DSL: https://github.com/jenkinsci/job-dsl-plugin/wiki
+
+import jobs.generation.*;
```

Comment:
Why do we have a new groovy file?  Does the new perf system just grab this instead?